src/api/app/controllers/webui/request_controller.rb

```diff
@@ -110,6 +112,23 @@ def show
     @history = History.find_by_request(@bsreq, {withreviews: 1})
     @actions = @req['actions']
 
+    # retrieve a list of all package maintainers what are assigned to at least one target package
+    @package_maintainers = get_target_package_maintainers(@actions)
+
+    # search for a project, where the user is not a package maintainer but a project maintainer and show
+    # a hint if that package has some package maintainers (issue#1970)
+    @show_project_maintainer_hint = false
+
+    unless @package_maintainers.empty?
+      @actions.each do |action|
+        target_package = Package.find_by_project_and_name(action[:tprj], action[:tpkg])
+        if target_package && User.current.is_maintainer_of_project?(action[:tprj]) && !@package_mainainers.include?(User.current)
```

Comment:
There is a typo here, you missed a t `!@package_mainainers.include?` should be `!@package_maintainers.include?`